samples/k8s-offer-azure-vote - custom/azure-vote-custom/values.yaml

```diff
@@ -0,0 +1,28 @@
+# Azure Vote UI control
+title: "Azure Vote Custom App"
+value1: Cats
+value2: Dogs
+getvalue: 'Get last one day usage' 
+global:
+  azure:
+    identity:
+      clientId: ""
+    marketplace:
+      planId: ""
```

Comment:
Specify as DONOTMODIFY?
Please add a comment that 'Azure populates these values at deployment time'